specification/hugr.md

```diff
@@ -155,9 +156,9 @@ As well as the type, dataflow edges are also parametrized by a
     [Non-local Edges](#non-local-edges)
 
 
-```
-SimpleType ::= ClassicType | LinearType
+`SimpleType` $\supset$ `ClassicType`
```

Comment:
![image](https://github.com/CQCL-DEV/hugr/assets/12997250/e837fcaa-1bae-4127-a441-c6ccfb73cf5e)


this looks to be free-floating, could just go in the ``` block below?

Reply:
Not if we want to keep that $\supset$...ok, I have used unicode. I'm mildly impressed that that actually renders in a monospaced font...so have included a comment for anyone viewing whose font doesn't.